datastore-v1-proto-client/src/test/java/com/google/datastore/v1/client/RemoteRpcTest.java

```diff
@@ -174,7 +174,7 @@ public void testHttpHeaders_expectE2eChecksumHeader() throws IOException {
         httpRequest
             .getHeaders()
             .getFirstHeaderStringValue(EndToEndChecksumHandler.HTTP_REQUEST_CHECKSUM_HEADER);
-    assertEquals(32, header.length());
```

Comment:
sorry I missed this in the previous review, can you confirm why this is changing and if it's expected?

Reply:
MD5 is 32 bytes long
since we are changing to CRC32c - which is 8 bytes long - this change reflects that..